PR: https://github.com/odoo/odoo/pull/178200

From: f10bd8ddcf99c9247dca417fefbc260454d74ea1
From: Gorash
Date: 2024-10-15 09:33:02

Breaking data model changes scores: del:5 + add:0, change matches:
-    _inherit = "stock.move"
-    _inherit = "stock.move.line"
-    _inherit = "stock.picking.type"
-    _inherit = "stock.picking"
-    _inherit = 'stock.warehouse'

Total Changes: 12

[IMP] *: adapt model class names to correspond to model names (apply script)

Part-of: odoo/odoo#178200
Related: odoo/enterprise#69762
Related: odoo/documentation#11115
Related: odoo/upgrade#6577
Related: odoo/design-themes#988
Related: odoo/upgrade-util#143

================================= pseudo patch: =================================

--- a/addons/stock_picking_batch/models/stock_move.py
+++ b/addons/stock_picking_batch/models/stock_move.py
@@ -6,7 +6,7 @@ from odoo.osv import expression
 
 
 class StockMove(models.Model):
-    _inherit = "stock.move"
+    _inherit = ["stock.move"]
 
     def _search_picking_for_assignation_domain(self):
         domain = super()._search_picking_for_assignation_domain()

--- a/addons/stock_picking_batch/models/stock_move_line.py
+++ b/addons/stock_picking_batch/models/stock_move_line.py
@@ -10,7 +10,7 @@ from odoo.tools.misc import OrderedSet
 
 
 class StockMoveLine(models.Model):
-    _inherit = "stock.move.line"
+    _inherit = ["stock.move.line"]
 
     batch_id = fields.Many2one(related='picking_id.batch_id', store=True)
 

--- a/addons/stock_picking_batch/models/stock_picking.py
+++ b/addons/stock_picking_batch/models/stock_picking.py
@@ -7,7 +7,7 @@ from odoo.exceptions import ValidationError
 
 
 class StockPickingType(models.Model):
-    _inherit = "stock.picking.type"
+    _inherit = ["stock.picking.type"]
 
     count_picking_batch = fields.Integer(compute='_compute_picking_count')
     count_picking_wave = fields.Integer(compute='_compute_picking_count')
@@ -84,7 +84,7 @@ class StockPickingType(models.Model):
 
 
 class StockPicking(models.Model):
-    _inherit = "stock.picking"
+    _inherit = ["stock.picking"]
 
     batch_id = fields.Many2one(
         'stock.picking.batch', string='Batch Transfer',

--- a/addons/stock_picking_batch/models/stock_picking_batch.py
+++ b/addons/stock_picking_batch/models/stock_picking_batch.py
@@ -8,9 +8,9 @@ from odoo.exceptions import UserError
 from odoo.osv.expression import AND
 from odoo.tools import float_is_zero, format_list
 
+
 class StockPickingBatch(models.Model):
     _inherit = ['mail.thread', 'mail.activity.mixin']
-    _name = "stock.picking.batch"
     _description = "Batch Transfer"
     _order = "name desc"
 

--- a/addons/stock_picking_batch/models/stock_warehouse.py
+++ b/addons/stock_picking_batch/models/stock_warehouse.py
@@ -5,7 +5,7 @@ from odoo import models
 
 
 class StockWarehouse(models.Model):
-    _inherit = 'stock.warehouse'
+    _inherit = ['stock.warehouse']
 
     def _get_picking_type_create_values(self, max_sequence):
         data, next_sequence = super()._get_picking_type_create_values(max_sequence)
